tests/LoggerTest.php

```diff
@@ -184,5 +184,11 @@ public function testAdapters(): void
          $logger = new Logger($adapter);
          $response = $logger->addLog($log);
          $this->assertEquals(200, $response);
+
+        // Test HoneyBadger
+        $adapter = new HoneyBadger(\getenv("TEST_HONEYBADGER_KEY"));
```

Comment:
The static analyzer is failing:

```
> ./vendor/bin/phpstan analyse --level max src tests
  0/11 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0% 11/11 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ -------------------------------------------------------------------- 
  Line   tests/LoggerTest.php                                                
 ------ -------------------------------------------------------------------- 
  189    Parameter #1 $configKey of class Utopia\Logger\Adapter\HoneyBadger  
         constructor expects string, string|false given.                     
 ------ -------------------------------------------------------------------- 


Error:  [ERROR] Found 1 error     
```

Would you please also fix the alignment on the logOwl lines above?

Reply:
@2002Bishwajeet, it looks like the static analyzer is still failing 🧐

Reply:
Should hopefully fix now